logger/logger.go

```diff
@@ -0,0 +1,24 @@
+package logger
+
+import (
+	"go.uber.org/zap"
+)
+
+// Logger is a wrapped logger used by the stanza agent.
+type Logger struct {
```

Comment:
Do we need a new type here? Or can we just create the logger with the new core and be done with it?